src/itemChoice/story.tsx

```diff
@@ -26,14 +28,14 @@ stories.add(
           labelInfo={text('Title Info', 'Additional informations')}
           data={text('Data', 'Data')}
           dataInfo={text('Data Info', 'Info')}
-          leftAddon={withLeftIcon ? <ComfortIcon /> : null}
-          rightAddon={withRightIcon ? <ComfortIcon /> : null}
+          leftAddon={withLeftIcon ? <CheckIcon isDisabled={disabled} /> : null}
```

Comment:
Too bad we have to pass the disabled state to the left/right addons :/
Not sure how to fix it easily though.

Reply:
Yes, I thought of checking if the ReactNode has a "disabled" prop and if so change it in the Item component but even so I'd rather try it in a separate branch